Lib/telnetlib.py

```diff
@@ -552,7 +552,18 @@ def interact(self):
                             print('*** Connection closed by remote host ***')
                             return
                         if text:
-                            sys.stdout.write(text.decode('ascii'))
+                            try:
+                                sys.stdout.write(text.decode('ascii'))
+                            except UnicodeDecodeError:
+                                if hasattr(sys.stdout, 'buffer'):
```

Comment:
The `hasattr(sys.stdout, 'buffer')` check is not required - I realized in the original PR you linked, it checked for Python 3. `sys.stdout.buffer` does indeed only exist in Python 3 and not in Python 2. However since Python 2 is no longer being actively maintained, there is no longer a need to check for `sys.stdout.buffer`, that check will always succeed anyways.

Reply:
That' true. I remove these checks